src/main/scala/utils/images/Image.scala

```diff
@@ -152,10 +152,12 @@ case class ImageMetadata(xDim: Int, yDim: Int, numChannels: Int)
 case class ByteArrayVectorizedImage(
     vectorizedImage: Array[Byte],
     override val metadata: ImageMetadata) extends VectorizedImage {
-  override def imageToVectorCoords(x: Int, y: Int, channelIdx: Int): Int = {
+  def imageToVectorCoords(x: Int, y: Int, channelIdx: Int): Int = {
     channelIdx + y*metadata.numChannels + x*metadata.yDim*metadata.numChannels
   }
 
+  def vectorToImageCoords(v: Int): Coordinate = ???
```

Comment:
What does the fact that this isn't implemented make ByteArrayVectorizedImage unusable for?